src/Connection.js

```diff
@@ -0,0 +1,52 @@
+
+define(["ObjectsConfig"], function (ObjectsConfig) {
+    "use strict";
+
+    var Factory = function () {
+        this.createConnection = function (url) {
+            var session = new ObjectsConfig.WebSocket(url);
+            var connection = new Connection(session);
+            return connection;
+        }
+    }
+
+    var Connection = function(session) {
```

Comment:
Should get a sessionFactory as parameter
Creating a session is the purpose of a connection object !